javascript/javascript2/week2/home.css

```diff
@@ -0,0 +1,118 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+}
+
+
+body {
+
+    background-color: #eef4ff;
+    color: #333;
+}
+
+/* Panel styles */
+.panel {
+    width: 80%;
+    max-width: 800px;
+    min-height: 40vh;
+    background-color: #fff;
+    margin: 20% auto;
+    border-radius: 10px;
+    padding: 30px;
+    border: 1px solid #eb8328;
+   
+}
```

Comment:
The panel appears in the center of the page, but very low down. Try and add some styling which will put it exactly in the center.